### PR TITLE
BoS Timelocks +2 Knight Slots for SK/K, Add Ripper & BPF

### PR DESCRIPTION
**Pull request by Cythisiax**
increases BoS timelocks nicely

Knight/Scribe: 10 hours
Senior: 30 hours
Head Knight/Scribe: 50 hours
Paladin: 40 hours
Head Paladin: 80 hours

Adds Ripper & BPF to JK/K and SK. 

+1 Knight slot
+1 Senior Knight slot

### CODE REVIEW

**Comment by AdamElTablawy** [comment]
approved

File: code/modules/jobs/job_types/bos.dm [hunk 1]
```diff
@@ -862,8 +871,17 @@ Knight
 		/obj/item/gun/energy/laser/pistol=1,
 		/obj/item/stock_parts/cell/ammo/ec=2,
 		)
-
+		
 /datum/outfit/loadout/knightc
+	name = "Junior Knight-Cavalry"
+	backpack_contents = list(
+		/obj/item/clothing/accessory/bos/juniorknight=1,
+		/obj/item/melee/powered/ripper/prewar,
```
Comment (OneOneThreeEight) [inline code/modules/jobs/job_types/bos.dm:879]: /obj/item/melee/powered/ripper/prewar = 1,

File: code/modules/jobs/job_types/bos.dm [hunk 2]
```diff
@@ -862,8 +871,17 @@ Knight
...
+		/obj/item/shield/riot/bullet_proof,
```
Comment (OneOneThreeEight) [inline code/modules/jobs/job_types/bos.dm:880]: /obj/item/shield/riot/bullet_proof = 1,

File: code/modules/jobs/job_types/bos.dm [hunk 3]
```diff
@@ -882,6 +900,14 @@ Knight
 		/obj/item/gun/energy/laser/pistol=1,
 		/obj/item/stock_parts/cell/ammo/ec=2,
 		)
+
+/datum/outfit/loadout/knightf
+	name = "Knight-Cavalry"
+	backpack_contents = list(
+		/obj/item/clothing/accessory/bos/knight=1,
+		/obj/item/melee/powered/ripper/prewar,
```
Comment (OneOneThreeEight) [inline code/modules/jobs/job_types/bos.dm:908]: /obj/item/melee/powered/ripper/prewar = 1,

File: code/modules/jobs/job_types/bos.dm [hunk 4]
```diff
@@ -882,6 +900,14 @@ Knight
...
+		/obj/item/shield/riot/bullet_proof,
```
Comment (OneOneThreeEight) [inline code/modules/jobs/job_types/bos.dm:909]: /obj/item/shield/riot/bullet_proof = 1,

File: code/modules/jobs/job_types/bos.dm [hunk 5]
```diff
@@ -778,29 +779,37 @@ Senior Knight
 		/obj/item/stock_parts/cell/ammo/ec=2,
 		)
 
-
+/datum/outfit/loadout/sknightd
+	name = "Cavalry"
+	backpack_contents = list(
+		/obj/item/clothing/accessory/bos/knight=1,
+		/obj/item/melee/powered/ripper/prewar,
```
Comment (OneOneThreeEight) [inline code/modules/jobs/job_types/bos.dm:786]: /obj/item/melee/powered/ripper/prewar = 1,

File: code/modules/jobs/job_types/bos.dm [hunk 6]
```diff
@@ -778,29 +779,37 @@ Senior Knight
...
+		/obj/item/shield/riot/bullet_proof,
```
Comment (OneOneThreeEight) [inline code/modules/jobs/job_types/bos.dm:787]: /obj/item/shield/riot/bullet_proof = 1,

**Review by OneOneThreeEight — CHANGES_REQUESTED** [review]

**Pull request merged**